admin/upgrade/3.x-to-4.x/index.rst

```diff
@@ -0,0 +1,104 @@
+.. _geonode-upgrade-3.x-4.x:
+
+========================================
+Upgrade Process from GeoNode 3.x to 4.x
+========================================
+
+The purpose of this document is to detail the process that we will implement to upgrade GeoNode from version 3.x to version 4.x. This document is proposed to ensure that the upgrade process is efficient and understandable for everyone. It is important to note that this guide is specifically for a GeoNode installation running on Docker.
+
+The objective of this document is for everyone to understand the processes and steps necessary to carry out the migration.
+
+Table of Contents
+
+* Migration Procedure
+* Possible Issues
+
+Version Migration Procedure
+===========================
+
+1. Download the New Version
+---------------------------
+
+With the following command, we will download the new version of GeoNode from the official source or the corresponding repository:
+
+::
+
+    https://github.com/GeoNode/geonode.git -b 4x
```

Comment:
I would use a placeholder in place of `4x` (which doesn't exist) and add a note about replacing the placeholder with either a branch or tag.

Reply:
Done